l10n_it_riba_sale_commission/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "ITA - Integrazione tra RiBa e provvigioni su vendite",
+    "summary": "Modulo ponte tra provvigioni agenti e RiBa",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Localization/Italy",
+    "website": "https://github.com/OCA/l10n-italy",
+    "author": "Nextev Srl, Odoo Community Association (OCA)",
+    "maintainers": ["odooNextev"],
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "sale_commission",
```

Comment:
Questa dipendenza è stata splittata in `15.0` (vedi https://github.com/OCA/commission/pull/387) in diversi moduli, ora dovrebbe essere sufficiente **account_commission**.

Reply:
Grazie, ero partito da un branch non aggiornato e quindi non ero correttamente allineato alle ultime modifiche. Il modulo dipende da https://github.com/OCA/commission/pull/561 che al momento è in attesa di reviews. Nei prossimi giorni provo a verificare quanto rimane da correggere su questa pr.